corehq/apps/enterprise/enterprise.py

```diff
@@ -241,23 +245,56 @@ def headers(self):
     def _query(self, domain_name):
         time_filter = form_es.submitted
 
-        users_filter = form_es.user_id(UserES().domain(domain_name).mobile_users().show_inactive()
-                                    .values_list('_id', flat=True))
+        users_filter = form_es.user_id(
+            UserES().domain(domain_name).mobile_users().show_inactive().size(self.MAXIMUM_USERS_PER_DOMAIN + 1)
+            .values_list('_id', flat=True)
+        )
 
-        query = (form_es.FormES()
-                 .domain(domain_name)
-                 .filter(time_filter(gte=self.datespan.startdate,
-                                     lt=self.datespan.enddate_adjusted))
-                 .filter(users_filter))
+        if len(users_filter) == self.MAXIMUM_USERS_PER_DOMAIN:
```

Comment:
Should this use `>` rather than `==`?

Reply:
Thanks for catching this. This looks like an off-by-one error, as I intended to check against `self.MAXIMUM_USERS_PER_DOMAIN` + 1, but using greater than is safer. Addressed in 45eb969c934db0da82bcbeb8aa1b23d1025ac835